OpenRA.Mods.Common/Activities/Air/Land.cs

```diff
@@ -251,5 +254,10 @@ public override bool Tick(Actor self)
 
 			return false;
 		}
+
+		public override IEnumerable<TargetLineNode> TargetLineNodes(Actor self)
+		{
+			yield return new TargetLineNode(target, targetLineColor.Value);
```

Comment:
Shouldn't this be checking `targetLineColor != null` first?

Reply:
Fixed.